bin/porta

```diff
@@ -475,7 +505,7 @@ module Porta
 
     def run
       queues_arg = queues.map { |queue| "--queue #{queue}" }
-      exec_in_porta('PROMETHEUS_EXPORTER_PORT' => '9395') { "bundle exec sidekiq #{queues_arg.join(' ')} -c #{concurrency}" }
+      exec_in_porta { "bundle exec sidekiq #{queues_arg.join(' ')} -c #{concurrency} --index 2" }
```

Comment:
I thought it looks a little nicer in this way but I can revert this as both have equivalent outcome.